embuilder.py

```diff
@@ -263,16 +279,14 @@ def is_flag(arg):
       build(CXX_WITH_STDLIB, ['libc++.a'], ['-s', 'DISABLE_EXCEPTION_CATCHING=0'])
     elif what == 'libc++_noexcept':
       build(CXX_WITH_STDLIB, ['libc++_noexcept.a'])
+    elif what == 'libc++-mt':
+      build(CXX_WITH_STDLIB, ['libc++.a'], ['-s', 'DISABLE_EXCEPTION_CATCHING=0', '-s', 'USE_PTHREADS=1'])
+    elif what == 'libc++-mt_noexcept':
+      build(CXX_WITH_STDLIB, ['libc++_noexcept.a'], ['-s', 'USE_PTHREADS=1'])
```

Comment:
It might be good to start transitioning to using `-pthread` instead of `USE_PTHREADS=1` (unless this has to work with fastcomp too?)

Reply:
It does need to work in fastcomp too, yes (although we could special-case things there and ignore the -mt versions as has been the case so far, but then comparing bugs between the two may get harder).